<launch>
    <node pkg="gps_optimize_tool" type="gps_distance" name ="GPS_optimize_trigger" output="screen"/>
</launch>